content/libraries/download.adoc

```diff
@@ -34,11 +36,7 @@ The set of libraries corresponding to `v4.xx` of KiCad software are organised on
 
 In conjunction with the KiCad v5 software release, the libraries will be reorganised into four separate repositories (on GitHub).
```

Comment:
In conjunction with the KiCad v5 software release, the libraries will be reorganised into four separate repositories (on GitHub). These repositories are currently a work in progress.